src/__utils/Collapse.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+export interface CollapseProps extends React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement> {
```

Comment:
same thing here, check my earlier comment about using `DetailedHTMLProps `

Reply:
if you go down to the return statement, you will find `{...props}` ... I am passing everything to the component